packages/std/src/ado_contract/execute.rs

```diff
@@ -145,6 +145,11 @@ impl<'a> ADOContract<'a> {
             }
         );
 
+        let permissioned_actions = self.query_permissioned_actions(deps.as_ref())?;
+        for action in permissioned_actions {
+            self.disable_action_permission(action, deps.storage);
```

Comment:
Does this still not have the same issue? If I permission the action again it'll be broken for certain addresses?

Reply:
Should be resolved now